### PR TITLE
pythonPackages.llfuse: 1.3.6 -> 1.3.8, fix tests

### LINKED ISSUE #106266: borgbackup: fails to build
**Describe the bug**
Borgbackup fails to build on nixos-unstable

**To Reproduce**
```bash
nix-shell -p pkgs.borgbackup 
```

**Expected behavior**
nix-shell with borgbackup

**Additional context**
This is the full error output:
https://gist.github.com/pinpox/dce57c63aa69eef6c489ea67e51a700d

**Notify maintainers**

@flokli @dotlambda @globin 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.81, NixOS, 21.03.git.83cbad92d73 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.9`
 - channels(root): `"nixos-21.03pre257339.83cbad92d73"`
 - channels(pinpox): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

### PR DESCRIPTION
**Pull request by erosennin**
###### Motivation for this change

Fixes #106266.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by Flakebi — APPROVED** [review]
Thanks, tested borg and it works fine.

Result of `nixpkgs-review pr 106279` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>s3ql</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>borgbackup</li>
    <li>python37Packages.llfuse</li>
    <li>python38Packages.llfuse</li>
    <li>python39Packages.llfuse</li>
  </ul>
</details>

`s3ql` was already broken, so not a regression.

**Comment by nyanloutre** [comment]
You may need to rebase on master and revert changes from this PR  #106270

**Comment by erosennin** [comment]
Thanks, done.

File: pkgs/development/python-modules/llfuse/default.nix [hunk 1]
```diff
@@ -8,30 +8,26 @@ in
...
   pname = "llfuse";
-  version = "1.3.6";
+  version = "1.3.8";
 
   src = fetchurl {
-    url = "mirror://pypi/l/llfuse/${pname}-${version}.tar.bz2";
-    sha256 = "1j9fzxpgmb4rxxyl9jcf84zvznhgi3hnh4hg5vb0qaslxkvng8ii";
+    url = "mirror://pypi/l/llfuse/${pname}-${version}.tar.gz";
+    sha256 = "1g2cdhdqrb6m7655qp61pn61pwj1ql61cdzhr2jvl3w4i8877ddr";
   };
```
Comment (jonringer) [inline pkgs/development/python-modules/llfuse/default.nix:16]: I would prefer to use pypi as tar.gz is the default extension
```
  src = fetchPypi {
    inherit pname version;
    sha256 = "1g2cdhdqrb6m7655qp61pn61pwj1ql61cdzhr2jvl3w4i8877ddr";
  };
```

**Comment by seppeljordan** [comment]
Result of `nixpkgs-review pr 106279` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>s3ql</li>
  </ul>
</details>
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>borgbackup</li>
    <li>python37Packages.llfuse</li>
    <li>python38Packages.llfuse</li>
    <li>python39Packages.llfuse</li>
  </ul>
</details>

**Comment by nyanloutre** [comment]
s3ql is getting fixed here : #106283

File: pkgs/development/python-modules/llfuse/default.nix [hunk 2]
```diff
@@ -1,5 +1,5 @@
-{ stdenv, fetchurl, fetchpatch, buildPythonPackage, pkgconfig, pytest_5, fuse, attr, which
-, contextlib2, osxfuse
+{ stdenv, fetchurl, fetchpatch, buildPythonPackage, pkgconfig, pytest, fuse, attr, which
```
Comment (jonringer) [inline pkgs/development/python-modules/llfuse/default.nix:1]: no longer needed
```suggestion
{ stdenv, fetchpatch, buildPythonPackage, pkgconfig, pytest, fuse, attr, which
```

**Comment by siriobalmelli** [comment]
LGTM

Result of `nixpkgs-review pr 106279` on Darwin:

4 packages built:
borgbackup python37Packages.llfuse python38Packages.llfuse python39Packages.llfuse

**Review by jonringer — APPROVED** [review]
LGTM

failures are broken on target branch
```
https://github.com/NixOS/nixpkgs/pull/106279
1 package failed to build:
s3ql

4 packages built:
borgbackup python37Packages.llfuse python38Packages.llfuse python39Packages.llfuse
```

**Pull request merged**